src/main/java/com/epam/repository/TourOfferDAO.java

```diff
@@ -20,7 +20,7 @@ public TourOfferDAO(JdbcTemplate JdbcTemplate) {
     }
 
     public List<TourOffer> getTours(){
-        return JdbcTemplate.query("SELECT * FROM tourOffer", (rs, rowNum) -> buildTour(rs));
+        return JdbcTemplate.query("SELECT * FROM tour_offer", (rs, rowNum) -> buildTour(rs));
```

Comment:
nope, you can get property from the property file

Reply:
done